src/transaction/log_page_buffer.c

```diff
@@ -1927,6 +1931,23 @@ logpb_copy_page (THREAD_ENTRY * thread_p, LOG_PAGEID pageid, LOG_CS_ACCESS_MODE
     }
 
   /* Could not get from log page buffer cache */
+
+#if defined (SERVER_MODE)
+  /* Send a request to Page Server for the log. */
+  if (get_server_type () == SERVER_TYPE_TRANSACTION)
+    {
+      char buffer[BIG_INT_SIZE];
+      memcpy (buffer, &pageid, sizeof (pageid));
```

Comment:
```suggestion
      std::memcpy (buffer, &pageid, sizeof (pageid));
```

Reply:
namespace "std" has no member "memcpy"C/C++(135)

Reply:
It is in the <cstring> header.